### PR TITLE
Fix javadoc errors

### PR DESCRIPTION
**Pull request by rodrigc**

### CODE REVIEW

**Comment by batmat** [comment]
:+1: 

I think the `Firstname Name <email>` pattern was on purpose and authors would have preferred keeping it replacing the \< by \&lt; entities and so on, but frankly I also don't think it's worth bothering :).

Thanks!

**Comment by batmat** [comment]
Also @rodrigc didn't test it myself, but I suppose you checked it actually fixes javadoc build issues?

File: core/src/test/java/hudson/model/FileParameterValueTest.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 /**
  * Test for {@link FileParameterValue}.
- * @author Oleg Nenashev <o.v.nenashev@gmail.com>
+ * @author <a href="mailto:o.v.nenashev@gmail.com">Oleg Nenashev</a>
```
Comment (oleg-nenashev) [inline core/src/test/java/hudson/model/FileParameterValueTest.java]: Uhh, I'd rather keep the name only.

File: core/src/test/java/jenkins/widgets/HistoryPageFilterTest.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public void test_latest_empty_page() {
     }
 
     /**
-     * Latest/top page where total number of items < the max page size.
+     * Latest/top page where total number of items greater than the max page size.
```
Comment (jtnord) [inline core/src/test/java/jenkins/widgets/HistoryPageFilterTest.java]: `<` is less than

**Comment by oleg-nenashev** [comment]
:+1: 
Maybe it makes sense to confirm with people that they are fine with hyperlink publishing.
It simplifies the work of spam recipient list crawlers.

**Comment by jtnord** [comment]
:-1: due to what appears to be an introduced documentation error

File: core/src/test/java/hudson/model/FileParameterValueTest.java (hunk 1 above)
Comment (rodrigc) [inline core/src/test/java/hudson/model/FileParameterValueTest.java]: @oleg-nenashev OK

**Comment by rodrigc** [comment]
@oleg-nenashev @jtnord I addressed your comments

**Comment by batmat** [comment]
:+1:

**Comment by batmat** [comment]
Ready IMO.

I'll just leave some time for others to do another quick review, since I missed the doc mistake last time so even I read more carefully this time there might still be missing something.

**Comment by jtnord** [comment]
:+1:

**Pull request merged**